docs-ui/components/doDont.tsx

```diff
@@ -0,0 +1,130 @@
+import {Component} from 'react';
+import {styled} from '@storybook/theming';
+
+import {IconCheckmark, IconClose} from 'app/icons';
+import theme from 'app/utils/theme';
+
+type BoxContent = {
+  text: string;
+  img: {
+    src: string;
+    alt: string;
+  };
+};
+type Props = {
+  boxes: {
+    do: BoxContent;
+    dont: BoxContent;
+  };
+};
+const DDWrapper = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin: 16px auto;
+  @media only screen and (max-width: ${theme.breakpoints[1]}) {
+    flex-wrap: wrap;
+    margin: 32px auto;
```

Comment:
Do we want to use `app/styles/space` for storybook components too?

Reply:
Yeah I think so